platform/throughput-report/index.md

```diff
@@ -45,18 +48,11 @@ The containers required for generating a usage report are:
 - [RavenDB](/servicecontrol/ravendb/containers.md)
   - Alternatively another RavenDB source can be used. The [connection string](/servicecontrol/servicecontrol-instances/deployment/containers.md#required-settings-ravendb-connection-string) must be supplied when installing ServiceControl
 - [ServiceControl](/servicecontrol/servicecontrol-instances/deployment/containers.md)
-
-> [!WARNING]
-> By default, the ServiceControl instance will immediately start reading from the `error` queue when it starts. If this is not desired, e.g. if you have another process that monitors the error queue, then one of the following parameters should be supplied to the `docker run` command
```

Comment:
This was pulled up, also this currently is very "docker cli" specific which is not ideal for Kubernetes.